docs/src/examples/components/Ref/Types/index.tsx

```diff
@@ -15,6 +15,15 @@ const RefTypesExamples = () => (
       }
       examplePath="components/Ref/Types/RefExampleRef"
```

Comment:
the name of this file also can be simplified - `RefExample` should be just enough

Reply:
Yep, I've seen that Stardust uses different naming for examples.